### ISSUE TITLE
Investigate interoperability testing with EU testbed

### ISSUE THREAD

**Issue by wyc** [spruceid/didkit#99]
Answer the following questions:
- What is being tested?
- How much is related to DIDKit and Credible?
- Where would this fit into our existing testbed?

https://joinup.ec.europa.eu/collection/interoperability-test-bed-repository/solution/interoperability-test-bed/news/itb-v1120-released

**Comment by sbihel** [comment]
> What is being tested?
My understanding is that it is a test framework for API conformance. You provide scenarios (e.g. requests, schemas for expected response) and it executes them for you.

> How much is related to DIDKit and Credible?
Nothing specific to these projects.

> Where would this fit into our existing testbed?
An example would be replacing the server used in the `vc-http-api` test suite. They also offer to host your tests.

----

All in all, to my understanding, I don't quite see the value in it. The standardisation of test execution is arguably a good thing, it could make it easier to test your service against specifications. But you have to give up your favourite language/framework and use their homemade DSL and system (which may be excellent, and fit complex scenarios with multiple services interacting together -- I haven't checked in details). And in practise I, a developer, don't have much trouble using standards' test suite (I'm thinking of `vc-http-api`, or JSONSchema for example) -- and I definitely don't think the GITB would improve this part.

tl;dr: I might not be visualising the big picture or the politics correctly -- I mainly saw "yet another" test framework that aims for the unification of tests execution, which hasn't been a problem in my own experience.
